masterlist.yaml

```
@@ -18868,6 +18868,30 @@ plugins:
         util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 14
 
+  - name: 'Ryn''s Farms.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/72305/'
+        name: 'Ryn''s Farms'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Skyrim Sewers'
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Skyrim Sewers.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Rift Watchtower NavFix'
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Riften Watchtower Nav Fix.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Ryn''s Standing Stones](https://www.nexusmods.com/skyrimspecialedition/mods/64969/)'
+          - '[Ryn''s Skyrim Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/73778/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s_Standing_Stones_Ryn''s_Farms_PATCH.esp")'
+    clean:
+      - crc: 0x1BE50A99
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164/)'
```

Clean checksums don't support Markdown, you can just copy & paste the report xEdit generates without touching it:

```suggestion
      - crc: 0x1BE50A99
        util: 'SSEEdit v4.0.4'
```